src/components/books/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Books = () => <h1>BookStore view</h1>;
```

Comment:
- [ ] The books component is expected to render a list of books using the Book component. For now, you can simply iterate a list of mock books.